insights/parsers/softnet_stat.py

```diff
@@ -0,0 +1,105 @@
+"""
+softnet_stat - ``/proc/net/softnet_stat``
+=========================================
+
+This parser parses the stats from network devices. These stats includes events
+per cpu(in row), number of packets processed i.e packet_process (first column),
+number of packet drops packet_drops (second column), time squeeze i.e net_rx_action
+performed time_squeeze(third column), cpu collision i.e collision occur while obtaining
+device lock while transmitting cpu_collision packets (Ninth column), received_rps
+number of times cpu woken up received_rps (tenth column), number of times reached flow
+limit count flow_limit_count (eleventh column).
+
+    The typical contents of ``cat /proc/net/softnet_stat`` file is as below::
+
+        00008e78 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
+        000040ee 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
+        0001608c 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
+        0000372f 00000001 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
+
+        Column-01: packet_process: Packet processed by each CPU.
```

Comment:
The column description listed here conflicts with that in line 6~11:

```
number of packets processed i.e packet_process (first column),
number of packet drops packet_drops (second column), time squeeze i.e net_rx_action
performed time_squeeze(third column), cpu collision i.e collision occur while obtaining
device lock while transmitting cpu_collision packets (Ninth column), received_rps
number of times cpu woken up received_rps (tenth column), number of times reached flow
limit count flow_limit_count (eleventh column).
```

Which is correct?